content/notero-item.ts

```diff
@@ -134,6 +135,26 @@ export default class NoteroItem {
     return Zotero.URI.getItemURI(this.zoteroItem);
   }
 
+  public getPDFURL(): string {
+    const zoteroAPIKey = getNoteroPref(NoteroPref.zoteroAPIKey);
+    const zoteroUserID = getNoteroPref(NoteroPref.zoteroUserID);
+    const attachmentIDs = this.zoteroItem
+      .getAttachments(false)
+      .slice()
+      // Sort to get largest ID first
+      .sort((a, b) => b - a);
+
+    for (const id of attachmentIDs) {
+      const attachment = Zotero.Items.get(id);
+      if (attachment.attachmentContentType == 'application/pdf') {
+        const pdfItemID = Zotero.URI.getItemURI(attachment).split('/').pop();
+        return `https://api.zotero.org/users/${zoteroUserID}/items/${pdfItemID}/file/view?key=${zoteroAPIKey}`;
```

Comment:
Zotero has a `getItemPath()` method that should make this easier. It also includes the user ID, so we don't need to have that as a separate pref. 🙂

```suggestion
        return `https://api.zotero.org/${Zotero.URI.getItemPath(attachment)}/file/view?key=${zoteroAPIKey}`;
```